libnmstate/_error.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+class NmstateError(Exception):
+    BUG = -1
+    NOT_SUPPORT = 95
+
+    _LOCALS = locals()
+
+    @property
+    def errno_str(self):
+        for errno_str in list(NmstateError._LOCALS.keys()):
+            if NmstateError._LOCALS[errno_str] == self.errno:
+                return errno_str
+        return "UNKNOWN_ERROR_NUMBER"
```

Comment:
How about using the empty string iinstead?

Reply:
I guess in that case, it might slightly hard for me to know the invalid error number.

Reply:
I am not sure I follow, `""` would mean that the error number is unknown... Btw. Edy will request to use single ticks for strings here.